modules/calib3d/src/fundam.cpp

```diff
@@ -490,12 +490,47 @@ static int run7Point( const Mat& _m1, const Mat& _m2, Mat& _fmatrix )
     double* fmatrix = _fmatrix.ptr<double>();
     int i, k, n;
 
+    Point2d m1c(0, 0), m2c(0, 0);
+    double t, scale1 = 0, scale2 = 0;
+    constexpr int count = 7;
```

Comment:
Please remove `constexpr` - use C++98 only => `const`

Reply:
Got it.